### ISSUE TITLE
What happened to the "Working with data" link / page(s)?

### ISSUE THREAD

**Issue by jjjohn538** [MicrosoftDocs/visualstudio-docs#7267]
The content of the previously listed link (in the list of links about getting started with C# and ASP.Net Core) is presented instead of "working with data". Where is the "data" content?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b2376dee-30e0-850f-7572-6d32b351be3f
* Version Independent ID: 9c06f47f-8195-4736-3575-fffe1c51f763
* Content: [Tutorial: Get started with C# and ASP.NET Core](https://docs.microsoft.com/en-us/visualstudio/get-started/csharp/tutorial-aspnet-core?view=vs-2019)
* Content Source: [docs/get-started/csharp/tutorial-aspnet-core.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/get-started/csharp/tutorial-aspnet-core.md)
* Product: **visual-studio-windows**
* Technology: **vs-ide-general**
* GitHub Login: @anandmeg
* Microsoft Alias: **meghaanand**

**Comment by WilliamAntonRohm** [comment]
@jjjohn538 -- thank you for your question.  I am investigating.

cc @anandmeg for awareness

**Comment by jjjohn538** [comment]
WilliamAntonRohm thank you for this comment. Following your suggestion, I did a bit more searching and conclude that a link on github should be fixed. Here is the part that I may not have made clear. My search for learning about ASP.Net Core started at the link:
https://github-wiki-see.page/m/dotnet-architecture/eShopOnWeb/wiki/Getting-Started-for-Beginners

This page shows the following bulleted links:

- Installing Visual Studio 2019
- Creating an ASP.NET Core Web App
- Working with data using EF Core
- Exposing a Web API
- Deploying and hosting the app in Azure

The 2nd and 3rd links currently point to the same page. Following your idea I searched for "Working with data using EF Core". I found this page that I believe should be the replace in the 3rd bullet above. Here it is:
https://docs.microsoft.com/en-us/dotnet/architecture/modern-web-apps-azure/work-with-data-in-asp-net-core-apps

I doubt that I have permission to fix it. I don't believe more research on your part is warranted. I'll close this for now. However, it should be fixed for future visitors to the github page.
@anandmeg
